pages/models.py

```diff
@@ -43,13 +48,24 @@
     ),
 )
 
+RENDERERS = deepcopy(DEFAULT_MARKUP_TYPES)
+for i, renderer in enumerate(RENDERERS):
+    if renderer[0] == 'markdown':
+        markdown_index = i
+
+RENDERERS[markdown_index] = (
+    'markdown',
+    cmarkgfm.github_flavored_markdown_to_html,
+    'Markdown'
+)
```

Comment:
```suggestion
RENDERERS = [
    x, cmarkgfm.github_flavored_markdown_to_html if x == 'markdown' else xx, xxx,
    for x, xx, xxx in deepcopy(DEFAULT_MARKUP_TYPES)
]
```

I don't know what to name these but maybe this would be more clear? idk